package.json

```diff
@@ -27,7 +27,6 @@
     "test:firefox:http2:ponyfill": "npm run test:firefox:http2:ponyfill --workspace main",
     "oldtest": "npm run oldtest --workspace main",
     "oldtesthttp2": "npm run oldtesthttp2 --workspace main",
-    "types": "tsc --build",
```

Comment:
Does this generate a problem? The version release today, does it still have problem with types?

Reply:
Just checked the version of today (`1.0.8`) and https://github.com/fails-components/webtransport/issues/233 still persists due to the path to `./dist/lib/index.d.ts` that is failing. 

A potential solution could be pointing `"types": "./dist/main/lib/index.d.ts"`, but I believe having both projects compiled under `dist` was not intentional. 

Below is a screenshot of both modules (`1.0.8`) after being installed via NPM (`@fails-components/webtransport` and `@fails-components/webtransport-transport-http3-quiche`), you can see both projects have the same output under `dist` folder, whereas they should only contain their own output:

<img width="309" alt="Screenshot 2023-12-31 at 12 43 12" src="https://github.com/fails-components/webtransport/assets/130494/ef9c4403-b3a7-4cb7-b576-18e0f61fdea7">

Reply:
Ok, the screenshots match my development environment.
Yes, it was not intentional.
But I had to trouble with the crossreferencing of the two packages in types.
Does a run of `tsc --build` at workspace level run fine with your changes?